deployments/common/image/base.yml

```diff
@@ -4,8 +4,8 @@ channels:
   - http://ssb.stsci.edu/astroconda
   - defaults
 dependencies:
-  - bqplot-image-gl
-  - bqplot
+  #- bqplot-image-gl==1.4.1
```

Comment:
We should consider reconciling these with common-scripts/jupyter-lab-extension-setup and common-env/jupyter.pip.   Some packages might move,  others may already be there and would just be deleted here.  The common Dockerfile does install-common on the base env so standard lab setup applies.

Reply:
Agreed.  I think for now though I'm not going to mess with things for webbinar-4.  In case I need to rebuild, I'd rather keep things as close to same as possible.